src/smt/witness_form.cpp

```diff
@@ -86,11 +86,12 @@ Node WitnessFormGenerator::convertToWitnessForm(Node t)
     {
       d_visited.insert(cur);
       curw = SkolemManager::getOriginalForm(cur);
-      // if its witness form is different
+      // if its original form is different
       if (cur != curw)
```

Comment:
Original form is recursive, so this is basically a test of whether cur contains skolems.  We can't use unpurified form here for the check since it only is given to the skolems themselves.